### DISCUSSION TITLE
QQ: Specific reason GPG not enabled in installation docs for RHEL et al.?

### DISCUSSION THREAD

**Discussion by RielDowner** [nginx/unit#1389] [General]
Hi,

I was just wondering if there was a specific reason that the install documentation has the GPG check disabled for the RHEL repo config?

I've added the below to my unit.repo and it allowed me to install without issue and seems a pretty reasonable thing to have enabled? :)

gpgkey=https://unit.nginx.org/keys/nginx-keyring.gpg
gpgcheck=1

**Comment by ac000** [comment]
Yes, it is a good idea in general.

@thresheek Is there a good reason for us not to recommend to enable gpg checks?

**Comment by thresheek** [comment]
No good reason.

**Comment by callahad** [comment]
Resolved in https://github.com/nginx/unit-docs/pull/163 - thank you!
